Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Keep golang version aligned with latest yocto release
-FROM golang:1.17.13-bullseye as builder
+FROM golang:1.22.2-bullseye as builder
```

Comment:
Same as in [the previous PR](https://github.com/mendersoftware/mender-artifact/pull/604/files#r1580639114) :smiley: